jobbergate-agent/charmcraft.yaml

```diff
@@ -0,0 +1,39 @@
+name: jobbergate-agent
+title: The Jobbergate Agent life-cycle manager.
+type: charm
+summary: Jobbergate Agent package life-cycle encapsulated in a charm
```

Comment:
```suggestion
summary: Jobbergate Agent package lifecycle encapsulated in a charm
```

Reply:
See my comment above. I'll change it to "life cycle".

Reply:
Resolved in commit e15fece.